charts/server/values.yaml

```diff
@@ -41,6 +41,34 @@ extraSecretNamesForEnvFrom:
   # - woodpecker-github-secret
   - woodpecker-secret
 
+woodpecker:
+  # -- enable metrics in woodpecker (and create secret with random token)
+  metrics: false
```

Comment:
Should there be a comment that `metrics: true` is required for prometheus and grafana to work?